_summaries/gropp2020implicit_2.md

```diff
@@ -1,19 +1,85 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+paper: gropp2020implicit
+author: # samatharhay
+score: # 7
 ---
 
-TODO: Summarize the paper:
 * What is the core idea?
+
+The core idea of the paper was to introduce a new paradigm built on implicit geometric regularization for computing high fidelity implicit neural representations directly from raw data scans.
```

Comment:
Might be good to talk about Eikonal equations here (top-left of page 3) since this is how this paper differs from the others. Functions that solve Eikonal equations automatically have unit normals (gradients).